sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobContainerProperties.java

```diff
@@ -1,13 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.storage.blob;
-
-import com.azure.storage.blob.implementation.models.ContainerGetPropertiesHeaders;
-import com.azure.storage.blob.models.LeaseDurationType;
-import com.azure.storage.blob.models.LeaseStateType;
-import com.azure.storage.blob.models.LeaseStatusType;
-import com.azure.storage.blob.models.PublicAccessType;
+package com.azure.storage.blob.models;
```

Comment:
These classes can be annotated with `@Immutable`. I had forgot to mention that on a few of the non-generated models I saw.